WcaOnRails/app/views/static_pages/documents.html.erb

```diff
@@ -61,5 +61,12 @@
         </div>
       </div>
     </div>
+    <div class="panel panel-default">
+      <div class="panel-heading heading-as-link">
+        <h4 class="panel-title">
+          <%= link_to icon("file-text", t('documents.codeofethics')), "/documents/Code of Ethics.pdf", class: "full-space" %>
```

Comment:
How do you feel about moving it up, so it's grouped with other single-file documents (i.e. below the Bylaws)?
I think that would be slightly more neat.

Reply:
I agree it should be with the other 2 single-file documents. Maybe now that we have 3 that they should be alphabetical?

Reply:
I was actually thinking that the Determination Letter should be below everything, but I can get on Board with putting the 3 single-file documents on top and ordering them alphabetically.

Reply:
I went ahead with that option:

![image](https://user-images.githubusercontent.com/26309166/55109025-29241f00-50d5-11e9-9746-8318d9637b4e.png)

Although now the "Motions" and "Minutes" look misplaced.

Reply:
because of the alphabetical order? I guess that won't work on most of the other locales either :)

I think it looks great